### ISSUE TITLE
Can we add --no-sandbox arg to puppeteer calls ?

### ISSUE THREAD

**Issue by pierreminiggio** [JonnyBurger/remotion#136]
Hello ! had this issue before on others of my projects in which I used puppeteer, so I usually add --no-sandbox arg on every puppeteer project I run. Is it possible to add that to remotion ?

Example : https://github.com/pierreminiggio/youtube-channel-links-crawler/blob/b079fdbeeee789b4b3b482c49082b41d96a0a1ac/src/scraper.js#L17

Trace :
![image](https://user-images.githubusercontent.com/35456349/108831680-c1c9bf00-75ca-11eb-8e16-dd591c849cce.png)
```console
npm run build

> remotion-template@1.0.0 build
> remotion render src/index.tsx Hello out.mp4 --overwrite

📦 (1/3) Bundling video...
[████████████████████████████████████████] 100%Error: Failed to launch the browser process!
[0223/113040.069371:ERROR:zygote_host_impl_linux.cc(90)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.


TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md

    at onClose (/var/www/cron/youtube-likes-recap-video-maker/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:193:20)
    at Interface.<anonymous> (/var/www/cron/youtube-likes-recap-video-maker/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:183:68)
    at Interface.emit (events.js:327:22)
    at Interface.close (readline.js:424:8)
    at Socket.onend (readline.js:202:10)
    at Socket.emit (events.js:327:22)
    at endReadableNT (internal/streams/readable.js:1327:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
npm ERR! code 1
npm ERR! path /var/www/cron/youtube-likes-recap-video-maker
npm ERR! command failed
npm ERR! command sh -c remotion render src/index.tsx Hello out.mp4 --overwrite

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-23T10_30_40_123Z-debug.log
```

**Comment by pierreminiggio** [comment]
PR Here : https://github.com/JonnyBurger/remotion/pull/137

**Comment by JonnyBurger** [comment]
Fixed by #137
